cmd/bnc-config/main.go

```diff
@@ -74,4 +78,34 @@ func main() {
 			log.Fatalf("unable to write config: %v", err)
 		}
 	}
+
+	// generate skeleton file for each mount
+	if settings.sklPath != "" {
+		// we'll output the list in the end
+		fallbacks := make([]string, 0)
+		successes := make([]string, 0)
+		t := time.Now().UTC().Unix() // skeleton needs a reference time for the installations
+		for _, m := range config.Mounts {
+			// note: skeleton() will return with generic header content when error occured
+			s, err := skeleton(m.Mark, set, t)
+			if err != nil {
+				fallbacks = append(fallbacks, m.Mark)
+			} else {
+				successes = append(successes, m.Mark)
+			}
+			err = os.WriteFile(filepath.Join(settings.sklPath, fmt.Sprintf("%s00NZL.SKL", m.Mark)), []byte(s), 0600)
+			if err != nil {
+				log.Fatalf("couldn't write skeleton file: %s", err)
+			}
+		}
+		if settings.summary != "" {
+			// output a summary info of skeleton generating
+			var output strings.Builder
+			output.WriteString("successes: " + fmt.Sprintln(successes))
+			output.WriteString("generic headers: " + fmt.Sprintln(fallbacks))
```

Comment:
suggestion(non-blocking): `strings.Join(successes, ", ")` might result in slightly more pleasant output.